google-cloud-error_reporting/README.md

```diff
@@ -29,7 +29,7 @@ or
 
     $ gcloud auth login
 
-Please see [[gcloud beta auth application-default login][] document for the difference between these commands.
+Please see [gcloud beta auth application-default login][] document for the difference between these commands.
```

Comment:
Is this a markdown link? (Same comment for similar, below.) If so, the target is empty, and should be parentheses (`()`) instead of brackets.